app/helpers/items_helper.rb

```diff
@@ -1,11 +1,16 @@
 module ItemsHelper
+
+  def item_condition_label(item)
+    content_tag(:span, item.condition, class: item_condition_class(item))
+  end
+
   def item_status_label(item)
-    content_tag(:span, item.status, class: item_label_class(item))
+    content_tag(:span, item.status, class: item_status_class(item))
   end
 
   def edit_item_button
     return nil if cannot? :edit, @item
-    return link_to 'Edit Item',
+    return link_to "Edit #{@item.name}",
```

Comment:
here you used `@item.name` but above you used `.item` i think this reinforces that the other one should be changed.